Dockerfile.centos

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2012-2016 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Dharmit Shah  - Initial implementation
+#   Mario Loriedo - Improvements
+#
+# To build it, run in the repository root:
+#  `docker build -t registry.centos.org/eclipse/che-server -f Dockerfile.centos .`
+#
+# To run it:
+#  docker run --net=host \
+#             --name che \
+#             -v /var/run/docker.sock:/var/run/docker.sock \
+#             -v /home/user/che/lib:/home/user/che/lib-copy \
+#             -v /home/user/che/workspaces:/home/user/che/workspaces \
+#             -v /home/user/che/storage:/home/user/che/storage \
+#             registry.centos.org/eclipse/che-server
+#           
+FROM registry.centos.org/centos/centos:latest
```

Comment:
My question about what you wanted to do with the pipeline makes sense.  I think the only fixes remaining are:
1. Adjust the Dockerfile so that it pulls from a local assembly that is built instead of downloading from nightly the assembly.
2. Comine the RUN commands into a single RUN command to remove layers.

Then we should be ok to merge.